utilities/converter/large_image_converter/__init__.py

```diff
@@ -103,15 +104,20 @@ def _generate_geotiff(inputPath, outputPath, **kwargs):
         'compression': 'lzw',
         'quality': 90,
         'predictor': 'yes',
+        'bigTiff': 'yes',
     }
-    predictor = {
+    yes_no = {
         'none': 'NO',
-        'horizontal': 'STANDARD',
-        'float': 'FLOATING_POINT',
+        'no': 'NO',
         'yes': 'YES',
     }
+    predictor = {
+        'horizontal': 'STANDARD',
+        'float': 'FLOATING_POINT',
+    }.update(yes_no)
     options.update({k: v for k, v in kwargs.items() if v not in (None, '')})
-    cmdopt = ['-of', 'COG', '-co', 'BIGTIFF=YES']
+    cmdopt = ['-of', 'COG']
+    cmdopt += ['-co', 'BIGTIFF=%d' % predictor[yes_no['bigTiff']]]
```

Comment:
I think you want `predictor[yes_no['bigTiff']]` to be `yes_no[options['bigTiff']]`.

Reply:
Whoops. This is what happens when I try to multitask and not run the code